### ISSUE TITLE
A11y: Violations in table :The WAI-ARIA attribute 'aria-selected' is not valid for the element <li> with WAI-ARIA role 'none'

### ISSUE THREAD

**Issue by EvaWiese** [carbon-design-system/carbon-components-vue#1241]
the table titles do not match the A11y requirements: 
The WAI-ARIA attribute 'aria-selected' is not valid for the element with WAI-ARIA role 'none'

## Detailed description
Code Snipped:
<button role="tab" aria-controls="userManagement" id="userManagement-link" eslint-disable-next-line="" tabindex="0" class="bx--tabs--scrollable__nav-link" aria-selected="true"> Users </button>

> Describe in detail the issue you're having.
 Accessibility checker reports the column title as a violation:   the table titles do not match the A11y requirements: 
The WAI-ARIA attribute 'aria-selected' is not valid for the element <li> with WAI-ARIA role 'none'
Seems to be a problem in the Carbon Vue library, at least I can reproduce it with the storybook: https://vue.carbondesignsystem.com/?path=/story/components-cvtabs--default

> Is this a feature request (new component, new icon), a bug, or a general issue?
bug
> Is this issue related to a specific component?
problem in the Carbon Vue library

> What did you expect to happen? What happened instead? What would you like to see changed?
remove or fix the A11y problem in the Carbon Vue library
> What browser are you working in?
FF

> What version of the Carbon Design System are you using?

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
Digital Business Automation;  yes the shipment for 2021.09
## Steps to reproduce the issue

1) go to the https://dbaoc-dev.blueworkscloud.com/dbaoc/portal/index.jsp#/ -> home
2) start the scan with the accessibility Checker
3) see the violations: The WAI-ARIA attribute 'aria-selected' is not valid for the element <li> with WAI-ARIA role 'none' 

## Additional information

- Screenshots or code
- 
![image](https://user-images.githubusercontent.com/88837124/129201996-08579b8a-edeb-4c31-aa7a-3e24896d181c.png)

- Notes
this issue was reported  and checked by developer JASPERSK@de.ibm.com also here https://jsw.ibm.com/browse/DBACLD-7193
